ntp.go

```diff
@@ -139,12 +144,23 @@ type Response struct {
 	Leap           LeapIndicator // server's leap second indicator; see RFC 5905
 }
 
+type QOption struct {
```

Comment:
Let's rename this QueryOptions. And add a comment that will be used for the documentation.  Something like:

```go
// QueryOptions contains the list of configurable options that may be used with the Query function.
```